### ISSUE TITLE
Handle auth inheritance when converting to Postman format

### ISSUE THREAD

**Issue by Zerak630** [rangav/thunder-client-support#503]
**Is your feature request related to a problem? Please describe.**
I created a requests' collection on Thunder & wanted to export it for Postman. After converting & importing it, i realized that authorization part is missing (I use Bearer token for my whole collection by inheriting it in requests).

**Describe the solution you'd like**
The embedded Postman converter can now convert collection's authorization.

**Implementation:**

In Postman I added authorization in collection scope, then export it and compare with the file converted from Thunder. It seems that this `auth` part below should be the missing part:

```
{
  /* Auth is same level as info part */
  "info": {
    "name": "My collection"
  },

  "auth": {
    "type": "bearer",
    "bearer": [
    {
      "key": "token",
      "value": "{{token}}",
      "type": "string"
    }]
  },
  /* Rest of JSON file */
}
```

I'd love to contribute if needed, but i think this feature is minor and may be easily added 😃

**Comment by rangav** [comment]
Thanks @Zerak630 for reporting the issue, will look into it.

**Comment by rangav** [comment]
This feature is now implemented and published to marketplace as release version `v1.14.0`

See all features and bugs fixed here
https://github.com/rangav/thunder-client-support/releases/tag/v1.14.0
